ts/src/bitget.ts

```diff
@@ -3849,16 +3849,19 @@ export default class bitget extends Exchange {
         this.checkRequiredSymbol ('fetchMyTrades', symbol);
         await this.loadMarkets ();
         const market = this.market (symbol);
-        if (market['swap']) {
-            throw new BadSymbol (this.id + ' fetchMyTrades() only supports spot markets');
-        }
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchOrderTrades', market, params);
```

Comment:
also, if symbol is required and bitget does not have conflicting ids we don't need this we can simply check the `market['type']`